src/components/ProductPage/ProductSpecs.js

```diff
@@ -101,29 +101,60 @@ const Variant = styled(`div`)`
   }
 `
 
+const PreOrder = styled(`div`)`
+  padding: 1rem;
+  background: #fafafa;
+  margin-bottom: 2rem;
+`
+
 const removeCareInstructions = desc =>
   desc.split(/Care Instructions/).slice(0, 1)
 
 const ProductSpecs = props => {
   const {
-    product: {title, description, descriptionHtml, productType, variants}
+    product: {
+      title,
+      description,
+      handle,
+      descriptionHtml,
+      productType,
+      variants
+    }
   } = props
 
   const hasVariants = variants.length > 1
-  const isPoster = productType === 'poster'
-
   const {price} = variants[0]
+  const isPoster = productType === 'poster'
+  const isForPreorder = handle === 'party-corgi-tank-top'
+  // calculate shipping date for corgi tank tops
+  const now = new Date()
+  const preSaleEndsDate = new Date('February 28, 2020') // sale ends on 2/28
```

Comment:
- 7 weeks + whatever days left until pre-sale closes on 2/28.  
- just display 7 weeks after 2/28 (product will be unavailable anyway)